src/test/java/com/bobocode/bibernate/integration/entity/Person.java

```diff
@@ -0,0 +1,25 @@
+package com.bobocode.bibernate.integration.entity;
+
+import com.bobocode.bibernate.annotation.Column;
+import com.bobocode.bibernate.annotation.Entity;
+import com.bobocode.bibernate.annotation.Id;
+import com.bobocode.bibernate.annotation.Table;
+import lombok.Data;
+import lombok.experimental.Accessors;
+
+@Table("persons")
+@Data
+@Entity
+@Accessors(fluent = true)
+public class Person {
```

Comment:
**(nit)**Why do we need another entity for tests? Is the Product not enough?

Reply:
One entity is not enough, because different tests manipulate Product records. For this reason, tests are not independen